poetry.lock

```diff
@@ -1,10 +1,16 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
+
 [[package]]
 name = "absl-py"
 version = "0.13.0"
 description = "Abseil Python Common Libraries, see https://github.com/abseil/abseil-py."
 category = "dev"
 optional = false
 python-versions = "*"
+files = [
+    {file = "absl-py-0.13.0.tar.gz", hash = "sha256:6953272383486044699fd0e9f00aad167a27e08ce19aae66c6c4b10e7e767793"},
```

Comment:
Just adding hashes. All changes are unrelated to the version of the libraries themselves.